src/core/session.js

```diff
@@ -427,6 +427,8 @@ export class Session {
   submissionIsNavigatable(form, submitter) {
     if (this.formMode == "off") {
       return false
+    } else if (this.submitterReferencesNavigatable(form, submitter)) {
+      return true
```

Comment:
Could this work as a recursive call to `submissionIsNavigatable(submitter.form)` (intentionally omitting the `submitter` argument) to re-use the existing code?

```suggestion
    } else if (this.elementIsNavigatable(submitter) && submitter.form instanceof HTMLFormElement) {
      return this.submissionIsNavgatable(submitter.form)
```